pkg/solana/config/multinode.go

```diff
@@ -0,0 +1,87 @@
+package config
+
+import "time"
+
+type MultiNode struct {
+	// TODO: Determine current config overlap https://smartcontract-it.atlassian.net/browse/BCI-4065
```

Comment:
while the config defaults are probably chain specific, the parameters themselves do not seem chain specific - ideally this could be extracted

Reply:
Sounds good, I'll create another PR to implement the default TOML configs for MultiNode. We can get that working first and then decide how we can extract things after.